src/integration-tests/bash/cleanup.sh

```diff
@@ -20,21 +20,34 @@
 #                   given lease on a failure.
 #
 #   SHARED_CLUSTER  Set this to true if you want cleanup to delete tiller
+#                   TBD tiller delete is disabled 
```

Comment:
Commenting so that we don't forget about this.  We need to see how else this flag is used.  If it really is all about deleting tiller than it was a strange name :)